packages/fhir-converter/src/templates/cda/Header.hbs

```diff
@@ -24,7 +24,7 @@
     		{{>References/Composition/custodian.hbs ID=(generateUUID (toJsonString this)) REF=(concat 'Organization/' (generateUUID (toJsonString this.custodian.assignedCustodian.representedCustodianOrganization)))}},            
         {{/if}}
 
-        {{#each participant}}
+        {{#each (toArray this.participant)}}
```

Comment:
Bug! 
If we only hade one participant it doesnt evaluate since cant do each on non array objects. Causing data loss. Likely have a half dozen more errors like this throughout mapper. issue added to master ticket

Reply:
good catch 🕵️